### ISSUE TITLE
Getting errors when moving cursor in script block of SFC

### ISSUE THREAD

**Issue by JohnCampionJr** [johnsoncodehk/volar#68]
Error like this pop up almost nonstop:

```
[Error - 11:05:48 PM] Request textDocument/documentHighlight failed.
  Message: Request textDocument/documentHighlight failed with message: Could not find source file: '/Users/John/Git/Vue3NETCoreAuthentication/src/ClientApp/src/areas/account/pages/ConfirmEmail.vue.__VLS_script.js'.
  Code: -32603 
```

That is the file I'm in, and otherwise, it seems to work fine.   v0.21.11 with TS Plugin on.   

Not sure how to help diagnose.

**Comment by johnsoncodehk** [comment]
can you try add `lang="ts"` to `<script>`, or add `"allowJs": true` to tsconfig?

**Comment by JohnCampionJr** [comment]
Yup.  I literally just figured that out :)

**Comment by johnsoncodehk** [comment]
I will fix it late because this bug can be avoided by user. (and should not publish menu version in one day :P)

**Comment by johnsoncodehk** [comment]
fixed in 0.21.12.
